### ISSUE TITLE
Should turning off the plane wave generator while paused clear the wave area?

### ISSUE THREAD

**Issue by samreid** [phetsims/wave-interference#213]
Noted during #207, Should turning off the plane wave generator while paused clear the wave area?

@arouinfar what do you think?

**Comment by arouinfar** [comment]
@samreid, not necessarily. While clearing the wave immediately would be consistent with the unpaused behavior, it might look a bit strange. It seems a bit more natural to clear the wave when the sim is unpaused (or stepped forward).

That said, I don't have a particularly strong opinion on this issue, so I'll defer to your best judgement @samreid.

**Comment by samreid** [comment]
It seems if the user unclicks the red plane wave generator button, the intention is to clear the waves, whether in play mode or pause mode.  It is also odd if the sim is paused and the user unclicks then clicks the red button--the waves restart and keep going.  I'd like to try clearing the wave area when the button is deselected when paused.

**Comment by samreid** [comment]
Proposed fix committed, @arouinfar can you please test?
